.env.example

```diff
@@ -13,4 +13,6 @@ TASK_OUTPUT_PATH=your_tasks_output_path
 # The remote name
 REPOSITORY_NAME=your_repository_name
 # The max number of concurrent workers
-MAX_WORKERS=8
\ No newline at end of file
+MAX_WORKERS=8
+# The GitHub repository to use if you are not using your own
+AUTOMATA_DEFAULT_REPOSITORY=emrgnt-cmplxty/automata
```

Comment:
What you are calling `AUTOMATA_DEFAULT_REPOSITORY` is currently `REPOSITORY_NAME` in the existing env

Reply:
We could consider renaming this if it is causing confusion

Reply:
Updated to reflect this